fsspec/implementations/tests/test_dbfs.py

```diff
@@ -150,10 +150,145 @@ def test_dbfs_read_range(dbfsFS):
 def test_dbfs_read_range_chunked(dbfsFS):
     dbfsFS.rm("/FileStore/large_file.txt")
     assert "/FileStore/large_file.txt" not in dbfsFS.ls("/FileStore/", detail=False)
-    content = b"This is a test\n" * (1 * 2**18) + b"For this is the end\n"
+    content = b"This is a test\n" * (1 * 2 ** 18) + b"For this is the end\n"
     with dbfsFS.open("/FileStore/large_file.txt", "wb") as f:
         f.write(content)
     assert "/FileStore/large_file.txt" in dbfsFS.ls("/FileStore", detail=False)
     assert dbfsFS.cat_file("/FileStore/large_file.txt", start=8) == content[8:]
     dbfsFS.rm("/FileStore/large_file.txt")
     assert "/FileStore/large_file.txt" not in dbfsFS.ls("/FileStore/", detail=False)
+
+
+@pytest.mark.vcr()
+def test_dbfs_write_pyarrow_non_partitioned(dbfsFS):
+    import pandas as pd
+    import pyarrow as pa
+    import pyarrow.parquet as pq
```

Comment:
This will fail if these are missing in the environment. Can we do without them?

Reply:
I have commented them, is it better to remove them?

Reply:
yes please